src/bluesearch/entrypoint/database/parse.py

```diff
@@ -77,6 +78,29 @@ def init_parser(parser: argparse.ArgumentParser) -> argparse.ArgumentParser:
         If it does not exist yet, a directory with this path is created.
         """,
     )
+    parser.add_argument(
+        "--match-filename",
+        type=str,
+        help="""
+        Parse only files with a name matching the given regular expression.
+        Ignored when 'input_path' is a path to a file.
+        """,
+    )
+    parser.add_argument(
+        "--recursive",
+        action="store_true",
+        help="""
+        Parse files recursively.
+        """,
+    )
+    parser.add_argument(
```

Comment:
I am not sure how much you like or dislike short options. It is your call.


```suggestion
    parser.add_argument(
        "-n",
```

I have seen people using this short option multiple times. I don't really know why "n" specifically:D. Anyway, examples:

* https://www.sphinx-doc.org/en/master/man/sphinx-apidoc.html
* https://git-scm.com/docs/git-clean

Reply:
Why not but `-n` is already used with another meaning in our entrypoints:

https://github.com/BlueBrain/Search/blob/615aa4f5097d321397070bb82f053f6eb91070d4/src/bluesearch/entrypoint/embeddings.py#L140-L146

Wouldn't it be confusing then?

Reply:
You decide:)

Reply:
Done in f586d3caaa441938b6784de5d31ad48cdeffb63f.

Reply:
Done in f586d3caaa441938b6784de5d31ad48cdeffb63f.